test/test_Gaussian/main.cpp

```diff
@@ -22,10 +22,9 @@ int main()
         {
             catch_exception = true;
```

Comment:
Almost there!
Please delete this line and `catch_exception` declaration some lines above as we are no longer using it.

After this, I merge the PR 🎉

Reply:
Sorry, forgot to remove them!